pkg/istio-agent/sds-agent.go

```diff
@@ -31,11 +31,10 @@ import (
 
 	"istio.io/istio/pilot/pkg/security/model"
 	"istio.io/istio/pkg/kube"
-
+	"istio.io/istio/security/pkg/credentialfetcher"
```

Comment:
Interfaces should go to istio.io/istio/pkg/security - separate from implementation.